presto-main/src/main/java/com/facebook/presto/cost/TableScanStatsRule.java

```diff
@@ -101,20 +102,4 @@ private OptionalDouble asDouble(Session session, Type type, Optional<Object> opt
                 .map(value -> toStatsRepresentation(metadata, session, type, value))
                 .orElseGet(OptionalDouble::empty);
     }
-
-    private Constraint<ColumnHandle> getConstraint(TableScanNode node, Expression predicate, Session session, Map<Symbol, Type> types)
-    {
-        DomainTranslator.ExtractionResult decomposedPredicate = DomainTranslator.fromPredicate(
-                metadata,
-                session,
-                predicate,
```

Comment:
Note: `predicate` was always `TRUE_LITERAL` here. This was noticed by @rschlussel2  in the `com.facebook.presto.cost.CoefficientBasedStatsCalculator.Visitor#visitTableScan` where this was copied from.